datastore/README.md

```diff
@@ -27,17 +30,67 @@ The code has been tested in the following environment:
 | [Python](https://www.python.org/) | 3.11 |
 | [grpcio-tools](https://grpc.io/docs/languages/python/quickstart/) | 1.56.2 |
 
-## Using docker compose to manage the service
+## Some examples of using docker compose to manage the service
+
+(**NOTE:** Any environment variables used on these examples are defined in a separate section below)
+
+### Check current status
+
+```text
+docker compose ps -a
+```
+
+### Start service in normal mode (rolling buffer of observations within latest 24H)
 
-Example commands:
+```text
+docker compose down --volumes
+docker compose build
+docker compose up -d
```

Comment:
Setting the default only in Go is gives lots of warnings from docker. Maybe better to recommend to always use a .env?

```
(venv) Lukass-MacBook-Pro:e-soh-datastore-poc phaf$ docker compose up -d
WARN[0000] The "DYNAMICTIME" variable is not set. Defaulting to a blank string. 
WARN[0000] The "LOTIME" variable is not set. Defaulting to a blank string. 
WARN[0000] The "HITIME" variable is not set. Defaulting to a blank string. 
WARN[0000] The "CLEANUPINTERVAL" variable is not set. Defaulting to a blank string. 
[+] Running 5/5
 ✔ Network datastore_default    Created                                                                                                                                                                                                                                                                   0.0s 
 ✔ Volume "datastore_ts-data"   Created       
 ```

Reply:
Yes, I also find this annoying. Will fix.